shopping_mall/backend/tests/conftest.py

```
@@ -0,0 +1,245 @@
+# ── RAG 서비스 목 ─────────────────────────────────────────────────────────────
+
+class FakeRAGService:
+    """retrieve / retrieve_multiple 결과를 제어 가능한 RAG 스텁."""
+
+    def __init__(self, results: dict[str, list[str]] | None = None):
+        # collection → 반환할 문서 리스트. 없으면 빈 리스트.
+        self._results: dict[str, list[str]] = results or {}
+
+    def retrieve(
+        self,
+        question: str,
```

Ruff (0.15.10)

[warning] Unused method argument: `question`

(ARG002)